### DISCUSSION TITLE
Text Wrapping

### DISCUSSION THREAD

**Discussion by x3rt** [SixLabors/ImageSharp.Drawing#266] [Q&A] [answered]
I am attempting to write text within squares in a grid, and wrap the text to make sure it doesn't overflow/overlap with other squares.
See the image for how it currently looks;
![image](https://user-images.githubusercontent.com/51767230/228834087-7669f53b-742b-449f-b720-61031892cc46.png)

My code is the following:
[Main Code](https://paste.emzi0767.com/v/7047182905162776577#tA1Ex4X2ZoMIslKxB4jX1Jafv6uekjvBDMgSsPK+aFo=)
[Methods](https://paste.emzi0767.com/v/7047183181820735489#Tu/A1oOs1r4sIV+ktppcfSjXk8s90BHxJac964Wmx7g=)
I tried setting the `WrappingLength` but it didn't change anything

**Comment by tocsoft** [comment, marked as answer]
i'm fairly certain we don't/can't break inside word to wrap them... try adding some spaces into your test text and see if that work for you.

**Reply by x3rt** [reply]
Adding spaces to the test text worked, but the actual text will very likely not contain any spaces.
I managed to remedy this by adding Zero-Width Spaces between the characters.
Thanks!

**Reply by JimBobSquarePants** [reply]
You can actually use `WordBreaking.BreakAll` https://docs.sixlabors.com/api/Fonts/SixLabors.Fonts.WordBreaking.html
